src/lib.rs

```diff
@@ -6,38 +6,47 @@
 #![allow(unused_imports)]
 #![deny(missing_docs)]
 
-#[macro_use] extern crate serde_derive;
-extern crate serde;
+#[cfg(feature="full-throttle")]
+#[macro_use]
+pub extern crate serde_derive;
```

Comment:
I'm not sure I agree with making these dependencies public. Let me think about this for a bit.

Reply:
Definitely. I think I'll remove it from this review and open an issue for discussion.